tests/acceptance/asset-tag-test.js

```diff
@@ -1,8 +1,6 @@
 var chai = require('chai');
-var defaultOptions = require('../fixtures/default-options');
```

Comment:
I'm guessing all the removals in this file were jshint errors as the vars weren't getting used?